src/components/Main/Main.tsx

```diff
@@ -0,0 +1,42 @@
+import { useContext, useMemo, useState } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { Todo } from '../../types/Todo';
+import { TodoList } from '../TodoList/TodoList';
+import { Status } from '../../enums/Status';
+
+export const Main = () => {
+  const { todos, updateTodos } = useContext(TodosContext);
+  const [isToggleAllChecked, setIsToggleAllChecked] = useState(true);
```

Comment:
The initial value of 'isToggleAllChecked' should be false, as initially, not all todos are completed.